modules/gdscript/gdscript_parser.h

```diff
@@ -1510,7 +1511,7 @@ class GDScriptParser {
 	EnumNode *parse_enum(bool p_is_abstract, bool p_is_static);
 	ParameterNode *parse_parameter();
 	FunctionNode *parse_function(bool p_is_abstract, bool p_is_static);
-	void parse_function_signature(FunctionNode *p_function, SuiteNode *p_body, const String &p_type);
+	void parse_function_signature(FunctionNode *p_function, SuiteNode *p_body, const String &p_type, const int p_signature_start);
```

Comment:
```suggestion
	void parse_function_signature(FunctionNode *p_function, SuiteNode *p_body, const String &p_type, int p_signature_start);
```

We generally do not use `const` when passing primitive types by value. Also, I don't really like adding a new parameter for such a workaround, but I guess it's better than `#ifdef`ing the parameter.

Reply:
I could not really find a workaround for this, as the `signature_start_pos` could not really be moved elsewhere. I could move this `signature_start_pos` inside the `functionNode` if this is more desired.